payments-core/src/main/java/com/stripe/android/model/SourceParams.kt

```diff
@@ -19,6 +19,22 @@ class SourceParams private constructor(
      */
     @SourceType val typeRaw: String,
 
+    /**
+     * The authentication `flow` of the source to create. `flow` is one of `redirect`, `receiver`,
+     * `code_verification`, `none`. It is generally inferred unless a type supports multiple flows.
+     *
+     * See [flow](https://stripe.com/docs/api/sources/create#create_source-flow)
+     */
+    var flow: Flow? = null,
```

Comment:
Initially I thought this is kotlin `flow` API support, could you add some comments here clarify?

Reply:
I have the property documented. Is there anything else you'd like me to add?